_raml/apps/products/examples/post_product_request.json

```diff
@@ -0,0 +1,10 @@
+{
+  "productNumber": "TP-0815",
+  "name": "Brand new product",
+  "shortDescription": "Awesome product",
+  "description": "This is a brand new product",
+  "manufacturer": "Awesome Products Company",
+  "isVisible": true,
```

Comment:
Please remove *isVisible*, we just decided to refactor this attribute and don't want to publish it now